### PR TITLE
major mods and reconfiguration of 3 articles re Private Operators

### PR DESCRIPTION
**Pull request by genwhittTTD**
- major mods and reconfiguration of three articles re Private Operators
- Correction of product name, Confidential Container > Confidential Containers, throughout.

### CODE REVIEW

File: docs/guides/integration-options-private-operator.md [hunk 1]
```diff
@@ -31,17 +41,19 @@ The participant must sign a contract (see [Account Setup](../getting-started/gs-
...
+If you choose to be a Private Operator, several implementation options are available. You can do any of the following:
 
-- [Nitro](https://aws.amazon.com/ec2/nitro/) Enclave (AWS)
-- Confidential space (GCP)
-- Confidential computing environment (Azure)
+- Use a cloud services setup. UID2 supports hosting UID2 in an <Link href="../ref-info/glossary-uid#gl-enclave">enclave</Link> on the following cloud service providers (medium level of effort to implement):
+  - [Nitro Enclave](https://aws.amazon.com/ec2/nitro/) from AWS
+  - [Confidential Space](https://cloud.google.com/confidential-computing#confidential-space), a confidential computing option from [Google Cloud](https://cloud.google.com/docs/overview/) Platform
+  - [Confidential Container](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers), a confidential computing option from Microsoft Azure
+- Use your own machines to generate and manage UID2s (greater level of effort to implement).
```
Comment (thomasm-ttd) [inline docs/guides/integration-options-private-operator.md:52]: We should remove this line - there is no way to run private operators locally

File: docs/guides/integration-options-private-operator.md [hunk 2]
```diff
@@ -31,17 +41,19 @@ The participant must sign a contract (see [Account Setup](../getting-started/gs-
...
-Every Private Operator runs in one of the following:
+If you choose to be a Private Operator, several implementation options are available. You can do any of the following:
 
-- [Nitro](https://aws.amazon.com/ec2/nitro/) Enclave (AWS)
-- Confidential space (GCP)
-- Confidential computing environment (Azure)
+- Use a cloud services setup. UID2 supports hosting UID2 in an <Link href="../ref-info/glossary-uid#gl-enclave">enclave</Link> on the following cloud service providers (medium level of effort to implement):
+  - [Nitro Enclave](https://aws.amazon.com/ec2/nitro/) from AWS
+  - [Confidential Space](https://cloud.google.com/confidential-computing#confidential-space), a confidential computing option from [Google Cloud](https://cloud.google.com/docs/overview/) Platform
+  - [Confidential Container](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers), a confidential computing option from Microsoft Azure
```
Comment (thomasm-ttd) [inline docs/guides/integration-options-private-operator.md:51]: [Confidential Containers] (plural)

File: docs/guides/integration-options-private-operator.md [hunk 3]
```diff
@@ -51,20 +63,37 @@ The basic workflow is as follows:
...
 
 Each supported Private Operator implementation must meet rigorous security standards. Some security points include:
 
-- The Private Operator runs in a hardware-based trusted execution environment (TEE) hosted by one of the supported cloud providers listed in [Private Operator Integration Options](#private-operator-integration-options).
+- The Private Operator runs in a hardware-based trusted execution environment (TEE) hosted by one of the supported cloud providers listed in [Hosting Options for Private Operators](#hosting-options-for-private-operators).
 - The Private Operator must complete an attestation process before accessing the information needed to process UID2s.
 - The information on S3 is encrypted at rest and also encrypted in transit through TLS. In addition, access is limited to only correctly authorized and attested Private Operators.
 - The information retrieved at startup is not stored locally at any point. It is only ever held in memory, and the Private Operator is running in a protected environment that makes it difficult for anyone running the Operator (such as an Administrator), as well as any external players, to see the data that's in memory.
```
Comment (thomasm-ttd) [inline docs/guides/integration-options-private-operator.md:76]: Suggest we add:
- The client data sent to the Operator (such as email addresses or phone numbers) are never stored by the Private Operator. They are only used to generate UIDs and then discarded.

File: docs/guides/integration-options-private-operator.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md:52]: Done, thx!

File: docs/guides/integration-options-private-operator.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md:51]: Done, thx. Fixed this instances... will fix other instances separately per my Slack message.

File: docs/guides/integration-options-private-operator.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md:76]: Thx! Updated to this:
The Private Operator never stores DII that is sent for processing (emails addresses and/or phone numbers). The data is only used within the enclave, to generate UID2s, and is discarded immediately after processing.

File: docs/guides/integration-options-private-operator.md [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
-sidebar_position: 02
 ---
 
 import Link from '@docusaurus/Link';
 
-# Private Operator Integrations
+# UID2 Private Operator Integration Overview
 
-A Private Operator is a private instance of the UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link>. This means that a specific entity hosts a private instance of the UID2 Operator, exclusively for their own use.
+Private Operators of UID2 send first-party <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> to a secure environment for translation, and control the destinations for those identifiers. A participant that chooses to become a Private Operator can generate and manage UID2s, running a UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> service in a private environment.
```
Comment (thomasm-ttd) [inline docs/guides/integration-options-private-operator.md:11]: Not sure about this paragraph. It implies that the Private Operator sends the DII out - which it doesn't. Maybe:

Participants that host their own Private Operator send first-party directly identifying information (DII) to their own, local UID2 Operator service, running in a private environment.

File: docs/guides/integration-options-private-operator.md [hunk 5]
```diff
@@ -51,33 +60,49 @@ The basic workflow is as follows:
...
 - The Private Operator must complete an attestation process before accessing the information needed to process UID2s.
 - The information on S3 is encrypted at rest and also encrypted in transit through TLS. In addition, access is limited to only correctly authorized and attested Private Operators.
 - The information retrieved at startup is not stored locally at any point. It is only ever held in memory, and the Private Operator is running in a protected environment that makes it difficult for anyone running the Operator (such as an Administrator), as well as any external players, to see the data that's in memory.
+- The Private Operator never stores <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> that is sent for processing (emails addresses and/or phone numbers). The data is only used within the enclave, to generate UID2s, and is discarded immediately after processing.
+
+## Private Operator Limitations
+
+There are a couple of limitations to Private Operator functionality:
+- Private Operators do not currently support generating the UID2 token on the client side.
```
Comment (thomasm-ttd) [inline docs/guides/integration-options-private-operator.md:78]: This should be: 
Private Operators do not currently support Client-side integration (with a link to the glossary item for Client-side integration).

File: i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-options-private-operator.md [hunk 6]
```diff
@@ -1,83 +1,108 @@
...
-sidebar_position: 02
 ---
 
 import Link from '@docusaurus/Link';
 
-# Private Operator Integrations
+# UID2 Private Operator Integration Overview
 
-Private Operator は、UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> のプライベートインスタンスです。つまり、特定の組織が、自分自身の使用のために UID2 Operator のプライベートインスタンスを独占的にホストしています。
+Private Operators of UID2 send first-party <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> to a secure environment for translation, and control the destinations for those identifiers. A participant that chooses to become a Private Operator can generate and manage UID2s, running a UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> service in a private environment.
```
Comment (thomasm-ttd) [inline i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-options-private-operator.md:11]: Not sure if this is generated, but this should match my suggested changes above

**Review by thomasm-ttd — COMMENTED** [review]
Some small comments

File: docs/guides/integration-options-private-operator.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md:11]: Good point thx. Done.

File: docs/guides/integration-options-private-operator.md (hunk 5 above)
Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md:78]: Done, thx.

File: i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-options-private-operator.md (hunk 6 above)
Comment (genwhittTTD) [reply i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-options-private-operator.md:11]: For now, it's a copy. Updated the copy. Thx.

File: docs/guides/integration-options-private-operator.md (hunk 4 above)
Comment (thomasm-ttd) [reply docs/guides/integration-options-private-operator.md]: This still says that Private Operators send first party data ...
Participants send their first party data to their own Private operators.

Comment (genwhittTTD) [reply docs/guides/integration-options-private-operator.md]: Hi @thomasm-ttd I'm not seeing this. What I had is what you suggested:
Participants that host their own Private Operator send first-party directly identifying information (DII) to their own, local UID2 Operator service, running in a private environment.
Not sure how to address this.
I've updated slightly to say this:
send **their own** first-party directly identifying information (DII) to their own, local UID2 Operator service
Is that any better?

**Review by thomasm-ttd — APPROVED** [review]

**Pull request merged**